src/redis-cli.c

```diff
@@ -8885,6 +8885,28 @@ static int getDbSize(void) {
     return size;
 }
 
+static int getDatabases(void) {
+    redisReply *reply;
+    int dbnum;
+
+    reply = redisCommand(context, "CONFIG GET databases");
+
+    if (reply == NULL) {
+        fprintf(stderr, "\nI/O error\n");
+        exit(1);
+    } else if (reply->type == REDIS_REPLY_ERROR) {
+        dbnum = 16;
```

Comment:
Why not keep the default at 20?

Reply:
20 looks like a random number that antirez picked, i am ok with keeping 20
but I would prefer to choose 16 (the default value of databases), although it might be a breaking change.

Reply:
OK, I take it back: it does not really matter.